test/Extensions/Consul.Tests/ConsulClusteringOptionsTests.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Orleans.Configuration;
+using Xunit;
+
+namespace Consul.Tests
+{
+    [TestCategory("Consul")]
+    public class ConsulClusteringOptionsTests
+    {
+        [Fact, TestCategory("Consul")]
```

Comment:
These tests should have the "BVT" or "Functional" category so that they are executed during CI runs

Reply:
Ok, done.